models/renalStructures_CECT_segmentation/docs/README.md

```diff
@@ -0,0 +1,124 @@
+
+
+## **Additional Usage Steps**
+
+#### Execute training:
+
+```
+python -m monai.bundle run training --meta_file configs/metadata.json --config_file configs/train.json
+```
+Expected result: finished, Training process started
+
+
+#### Execute training with finetuning
+```
+python -m monai.bundle run training --dont_finetune false --meta_file configs/metadata.json --config_file configs/train.json
+```
+Expected result: finished, Training process started, model variables are restored
+
+#### Execute validation:
+
+##### Download validation data:
+```
+cd models/renalStructures_CECT_segmantation
+python -m monai.bundle run download_data --meta_file configs/metadata.json --config_file "['configs/train.json', 'configs/evaluate.json']"
+```
+Expected result: finished, `data/` folder is created and filled with images.
+
+#####  Run validation script:
+```
+python -m monai.bundle run evaluate --meta_file configs/metadata.json --config_file "['configs/train.json', 'configs/evaluate.json']"
+```
+Expected result: finished, `Key metric: val_mean_dice best value: 0.7844631671905518 at epoch: 1` is printed.
+
+## **System Configuration**
+The model was trained for 10000 epochs on 2 RTX2080Ti GPUs with [SmartCacheDataset](https://docs.monai.io/en/stable/data.html#smartcachedataset). This takes 1 days and 2 hours, with 4 images per GPU.
+Training progress is available on [tensorboard.dev](https://tensorboard.dev/experiment/VlEMjLdURH6SyFp216dFBg)
+
+
+## **Limitations**
+For developmental purposes only and cannot be used directly for clinical procedures.
+
+## **Citation Info**
+```
+@article{chernenkiy2023segmentation,
+  title={Segmentation of renal structures based on contrast computed tomography scans using a convolutional neural network},
+  author={Chernenkiy, IМ and Chernenkiy, MM and Fiev, DN and Sirota, ES},
+  journal={Sechenov Medical Journal},
+  volume={14},
+  number={1},
+  pages={39--49},
+  year={2023}
+}
+```
+
+## **References**
+
+[1] Myronenko, A. (2019). 3D MRI Brain Tumor Segmentation Using Autoencoder Regularization. In: Crimi, A., Bakas, S., Kuijf, H., Keyvan, F., Reyes, M., van Walsum, T. (eds) Brainlesion: Glioma, Multiple Sclerosis, Stroke and Traumatic Brain Injuries. BrainLes 2018. Lecture Notes in Computer Science(), vol 11384. Springer, Cham. https://doi.org/10.1007/978-3-030-11726-9_28
+
+[2] Chernenkiy, I. М., et al. "Segmentation of renal structures based on contrast computed tomography scans using a convolutional neural network." Sechenov Medical Journal 14.1 (2023): 39-49.https://doi.org/10.47093/2218-7332.2023.14.1.39-49
+
+#### **Tests used for bundle checking**
```

Comment:
Thanks for adding this section. I will run these checks one by one locally.
In addition, we added unit tests for a part of bundles recently to ensure config files are correct. For example:
https://github.com/Project-MONAI/model-zoo/blob/6122ec9475c87726e14ea0e7a06ffdc3532a1ea2/ci/unit_tests/test_brats_mri_axial_slices_generative_diffusion.py
Are you interested in preparing the tests on this bundle? Feel free to ask if you meeting any issues.
TH
Please note that this is not a hard requirement of contributing a bundle, and we can also merge this PR first and then add unit tests in a separate PR in the future.

Reply:
Hello! Sure, I can add bundle test, but I can do it on next week